main.py

```diff
@@ -1,9 +1,185 @@
-print("Wyświetlenie listy zrobionych zadań: 1")
-print("Wyświetlenie listy zadań do wykonania: 2")
-print("Dodaj zadanie: 3")
-while True:
-    odpowiedz = input('Wybierz menu ')
-    if odpowiedz == "3":
-        nowe_zadanie = input("Napisz jakie zadanie chcesz dodać: ")
-        print(nowe_zadanie)
-        break
\ No newline at end of file
+import sqlite3
+from sqlite3 import Error
+from task import task
+
+task_list = []
+database = r"C:\Users\mateusz.saganowski\PycharmProjects\pythonProject\pythonsqlite.db"
+
+
+def create_connection(db_file):
+    """ create a database connection to the SQLite database
+        specified by db_file
+    :param db_file: database file
+    :return: Connection object or None
+    """
+    conn = None
+    try:
+        conn = sqlite3.connect(db_file)
+        return conn
+    except Error as e:
+        print(e)
+
+    return conn
+
+
+def create_table(conn, create_table_sql):
+    """ create a table from the create_table_sql statement
+    :param conn: Connection object
+    :param create_table_sql: a CREATE TABLE statement
+    :return:
+    """
+    try:
+        c = conn.cursor()
+        c.execute(create_table_sql)
+    except Error as e:
+        print(e)
+
+
+def create_task(conn, task):
+    """
+    Create a new task
+    :param conn:
+    :param task:
+    :return:
+    """
+
+    sql = ''' INSERT INTO tasks(name,status)
+              VALUES(?,?) '''
+    cur = conn.cursor()
+    cur.execute(sql, task)
+    conn.commit()
+    return cur.lastrowid
+
+
+def main_db_upload(upload):
+    # create a database connection
+    conn = create_connection(database)
+
+    sql_create_tasks_table = """CREATE TABLE IF NOT EXISTS tasks (
+                                        id integer PRIMARY KEY,
+                                        name text NOT NULL,
+                                        status bool NOT NULL
+                                    );"""
+    # create tables
+    if conn is not None:
+
+        # create tasks table
+        create_table(conn, sql_create_tasks_table)
+    else:
+        print("Error! cannot create the database connection.")
+    with conn:
+        # create tasks
+        new_task = (upload.name, upload.status)
```

Comment:
nazwa new_task jest użyta jako funkcja. Zmień albo nazwę zmiennej albo funkcji (np. funkcji na create_new_task)

Reply:
zgadza się. Po jakimś czasie mogę zapomnieć które new_task jest do której funkcji. poprawiono

Reply:
Tzn tutaj to by nic nie robiło, ale w przyszłości może sprawić kłopoty, jakbyś chciał przekazać gdzie funkcję, a nie ją wywołać. (użyć bez nawiasów) okazałoby się że masz 2 takie nazwy i nie wiadomo której miałby użyć - zmiennej czy funkcji.
Jak CI się nudzi to możesz zerknąć: https://treyhunner.com/2020/01/passing-functions-as-arguments/
Ale to już bardziej zaawansowany temat więc możesz olać.